src/librustc/hir/map/mod.rs

```diff
@@ -439,25 +423,19 @@ impl<'hir> Map<'hir> {
     /// Returns the `NodeId` that corresponds to the definition of
```

Comment:
This should say `HirId` (for good measure, check the doc comments of all functions with changed signatures).